app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/testItemDetails/itemHeader/itemHeader.jsx

```diff
@@ -58,6 +60,12 @@ export const ItemHeader = ({
   const defectTypes = useSelector(defectTypesSelector);
   const getLogItemLink = useSelector(getLogItemLinkSelector);
   const link = getLogItemLink(item);
+  const { trackEvent } = useTracking();
+  const MAKE_DECISION_MODAL_EVENTS = getMakeDecisionModalEvents();
+
+  const onClickIssue = (pluginName) => {
+    trackEvent(MAKE_DECISION_MODAL_EVENTS.getOnClickIssueTicketEvent(pluginName));
+  };
```

Comment:
The parent of this component - `TestItemDetails` has an `eventsInfo` prop. Can we receive this event from it?